studio/schemas/documents/lesson.js

```diff
@@ -103,6 +103,49 @@ export default {
         layout: 'radio',
       },
     },
+    {
+      name: 'isCommunityResource',
+      title: 'Community Resource?',
+      type: 'boolean',
+    },
+    {
+      name: 'thumbnailUrl',
+      title: 'Thumbnail URL',
+      type: 'url',
+    },
+    {
+      name: 'iconUrl',
+      title: 'Icon URL',
+      type: 'url',
+    },
+    {
+      name: 'repoUrl',
+      title: 'Repo URL',
+      type: 'url',
+    },
+    {
+      name: 'codeUrl',
+      title: 'Code URL',
+      type: 'url',
+    },
+    {
+      name: 'eggheadRailsCreatedAt',
+      title: 'egghead Rails Created At',
+      description: 'Date this lesson resource was created on egghead.io',
+      type: 'datetime',
+    },
+    {
+      name: 'eggheadRailsUpdatedAt',
+      title: 'egghead Rails Updated At',
+      description: 'Date this lesson resource last updated on egghead.io',
+      type: 'datetime',
+    },
+    {
+      name: 'eggheadRailsPublishedAt',
+      title: 'egghead Rails Published At',
+      description: 'Date this lesson resource was published on egghead.io',
+      type: 'datetime',
+    },
```

Comment:
The reason we added fields for these timestamp values is that they represent useful metadata that we wouldn't otherwise have in Sanity.

For example, we display when a lesson was last updated on the lesson page. That is based on the `updated_at` field. If a lesson was updated 2 months ago, but we import it from egghead-rails to Sanity today, we don't want to go off Sanity's internal updated attribute, we want to know when it was last actually updated in a meaningful way by an editor/publisher.

Perhaps it would be better to instead add a field called something like _Displayed Updated At_ (or _Last Updated_). That'd make it a little clearer what it is used for and keep it separate from the _internal_ timestamp values. It doesn't need to be named after `eggheadRails...`, we'd just use the value from egghead-rails when first importing it. 🤔